subsys/net/lib/config/init.c

```diff
@@ -553,17 +553,8 @@ int net_config_init_app(const struct device *dev, const char *app_info)
 	/* This is activated late as it requires the network stack to be up
 	 * and running before syslog messages can be sent to network.
 	 */
-	if (IS_ENABLED(CONFIG_LOG_BACKEND_NET) &&
-	    IS_ENABLED(CONFIG_LOG_BACKEND_NET_AUTOSTART)) {
-		const struct log_backend *backend = log_backend_net_get();
-
-		if (!log_backend_is_active(backend)) {
-			if (backend->api->init != NULL) {
-				backend->api->init(backend);
-			}
```

Comment:
I think the idea here was that if the logging is not yet initialized, we need to do it here.

Reply:
this is part of the https://github.com/zephyrproject-rtos/zephyr/pull/89275. When CONFIG_LOG_BACKEND_NET_AUTOSTART is enabled, than the log backend net has the autostart flag set and will be automatically init by the logging subsystem